source/renderer/app/api/utils/localStorage.js

```diff
@@ -95,10 +96,37 @@ export default class LocalStorageApi {
     } catch (error) {} // eslint-disable-line
   });
 
+  getDataLayerMigrationStart = () => new Promise((resolve, reject) => {
```

Comment:
@daniloprates please replace with `getDataLayerMigrationAcceptance` - and use the same pattern `Start` -> `Acceptance` in all the other parts.